Resources/Prototypes/_SpaceCats/Catalog/electromouse_catalog.yml

```
@@ -0,0 +1,129 @@
+    StolenEnergy: 50
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 1
+
+- type: listing
+  id: ElectromDash
+  name: Рывок
+  description: Телепортирует вас на пару клеток вперед.
+  productAction: ActionElectroDash
+  categories:
+  - ElectromouseAbilities
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)